purescript.cabal

```diff
@@ -42,7 +42,7 @@ library
                    aeson >= 0.8 && < 0.9,
                    vector -any,
                    bower-json >= 0.7,
-                   aeson-better-errors >= 0.7,
+                   aeson-better-errors >= 0.8,
```

Comment:
This version bump is for the `throwCustomError` function, which I realised should be added to aeson-better-errors while I was writing this code, so I just added it and released a new version